chapter_preliminaries/probability.md

```diff
@@ -1,304 +1,750 @@
 
-This simple equation has rather profound implications as it allows us to reverse the order of conditioning. For instance, given the prevalence of symptoms for a given disease, the overall prevalence of diseases and of symptoms respectively, it allows us to infer the probability of a disease, given the manifestation of certain symptoms. This is  what a physician would do when diagnosing a patient. 
-
-In some cases we might not have direct access to $P(B)$, such as the prevalence of symptoms. In this case a simplified version of Bayes' rule comes in handy:
+This simple equation has profound implications because
+it allows us to reverse the order of conditioning.
+If we know how to estimate $P(B|A)$, $P(A)$, and $P(B)$,
+then we can estimate $P(A|B)$. 
+We often find it easier to estimate one term directly 
+but not the other and Bayes' theorem can come to the rescue here.
+For instance, if we know the prevalence of symptoms for a given disease,
+and the overall prevalences of the disease and symptoms, respectively,
+we can determine how likely someone is 
+to have the disease based on their symptoms.
+In some cases we might not have direct access to $P(B)$, 
+such as the prevalence of symptoms. 
+In this case a simplified version of Bayes' theorem comes in handy:
 
 $$P(A|B) \propto P(B|A) P(A)$$
 
-Since we know that $P(A|B)$ must be normalized to $1$, i.e., $\sum_a P(A=a|B) = 1$, we can use it to compute
+Since we know that $P(A|B)$ must be normalized to $1$, i.e., $\sum_a P(A=a|B) = 1$,
+we can use it to compute
 
 $$P(A|B) = \frac{P(B|A) P(A)}{\sum_b P(B=b|A) P(A)}.$$
 
-In statistics $P(A)$ is typically referred as a *prior*, $P(B|A)$ as the likelihood of $B$ occurring, given $A$, and $P(A|B)$ as the *posterior* probability of $A$, given $B$, in short, *the posterior*. The above equation states that an unnormalized posterior can be normalized easily. This has many useful applications. For instance, in a speech recognition model $A$ might be the text and $B$ might be the audio file. There we can compute the (normalized) posterior probability for a specific string by summing over all possible strings and looking at their relative weight. 
+In Bayesian statistics, we think of an observer 
+as possessing some (subjective) prior beliefs
+about the plausibility of the available hypotheses 
+encoded in the *prior* P(H),
+and a *likelihood function* that says how likely 
+one is to observe any value of the collected evidence 
+for each of the hypotheses in the class $P(E|H)$.
+Bayes' theorem is then interpreted as telling us
+how to update the initial *prior* P(H)
+in light of the available evidence E
+to produce *posterior* beliefs 
+$P(H|E) = \frac{P(E|H) P(H)}{P(E)}$.
+Informally, this can be stated as 
+"posterior equals prior times likelihood, divided by the evidence".
+Now, because the evidence $P(E)$ is the same for all hypotheses,
+we can get away with using the simplified form
+and simply normalize over the hypotheses.
```

Comment:
"the simplified form" in a bit unclear.
How about:
"we can get away with simply normalizing over the hypotheses".

Reply:
sure